### PR TITLE
Fix for REGISTRY-3539

### PR DESCRIPTION
**Pull request by Malintha**

### CODE REVIEW

**Comment by thushara35** [comment]
Please test this in a rc1 pack

**Comment by Malintha** [comment]
Tested and works fine. Even though It'll go with 5.3.0

**Comment by Malintha** [comment]
Good to go with 5.2.0

**Comment by thushara35** [comment]
merged, please close the associated jira and all.

File: modules/es-extensions/publisher/asset/wadl/themes/default/partials/view-asset-details-container.hbs [hunk 1]
```diff
@@ -41,13 +41,16 @@
...
                             <h2 class="field-title">
-                                <a class="collapsing-h2" data-toggle="collapse" area-expanded="true">
-                                    {{t "Associations"}}
+                                <a class="collapsing-h2 collapsed" data-toggle="collapse" area-expanded="true" href="#collapsedependencies">
+                                    <i class="cu-btn-exp-col btn-collapsed">{{t "Associations"}}</i>
                                 </a>
                             </h2>
 
+                            <div id="collapsedependencies" class="collpase">
+                            <div id=""
```
Comment (thushara35) [inline modules/es-extensions/publisher/asset/wadl/themes/default/partials/view-asset-details-container.hbs]: Please comment the fixed commit.

**Pull request merged**